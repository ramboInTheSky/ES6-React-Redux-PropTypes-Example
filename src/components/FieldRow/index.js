import styled from 'styled-components';

export default styled.div`
  clear: both;
  margin-bottom: 30px;
  max-width: 495px;
`;
