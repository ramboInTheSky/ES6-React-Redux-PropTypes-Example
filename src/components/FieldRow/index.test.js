import React from 'react';
import { shallow } from 'enzyme';

import 'jest-styled-components';

import FieldRow from './';

describe('FieldRow', () => {
  it('should render correctly', () => {
    expect(shallow(<FieldRow />)).toMatchSnapshot();
  });
});
