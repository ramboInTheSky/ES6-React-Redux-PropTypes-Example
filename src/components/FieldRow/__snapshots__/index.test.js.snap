// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FieldRow should render correctly 1`] = `
.c0 {
  clear: both;
  margin-bottom: 30px;
  max-width: 495px;
}

<div
  className="c0"
/>
`;
